tgui/packages/tgui/interfaces/Phosphorizer.js

```diff
@@ -0,0 +1,102 @@
+import { useBackend } from '../backend';
+import { Box, Button, ColorBox, Divider, Knob, NoticeBox } from '../components';
+import { Window } from '../layouts';
+
+export const Phosphorizer = (props, context) => {
+  const { data } = useBackend(context);
+  const { busy } = data;
+  return (
+    <Window
+      title="Phosphorizer"
+      width={360}
+      height={190}>
+      <Window.Content>
+        { busy ? (
+          <BusyWindow />
+        ) : (
+          <OperateWindow />
+        )}
+      </Window.Content>
+    </Window>
+  );
+};
+
+const OperateWindow = (props, context) => {
+  const { act, data } = useBackend(context);
+  const { lights, online, hostR, hostG, hostB } = data;
+
+  return (
+    <Box>
+      <NoticeBox success>
+        Please insert only standard light tubes and bulbs.
+      </NoticeBox>
+      <Box>
+        <strong># Loaded Lights:</strong>
+        {' '}{ lights }
+      </Box>
+      <Box>
+        Color Tuning:
+        {' '}
+        <Knob
+          inline
+          minValue={20}
+          maxValue={255}
+          value={hostR}
+          color="red"
+          format={value => "R:" + value}
+          onDrag={(e, value) => act('tune_hue', {
+            hue: "R",
+            output: value,
+          })}
+        />
+        <Knob
+          inline
+          minValue={20}
+          maxValue={255}
+          value={hostG}
+          color="green"
+          format={value => "G:" + value}
+          onDrag={(e, value) => act('tune_hue', {
+            hue: "G",
+            output: value,
+          })}
+        />
+        <Knob
+          inline
+          minValue={20}
+          maxValue={255}
+          value={hostB}
+          color="blue"
+          format={value => "B:" + value}
+          onDrag={(e, value) => act('tune_hue', {
+            hue: "B",
+            output: value,
+          })}
+        />
+        {' '}
+        <ColorBox color={`rgba(${hostR}, ${hostG}, ${hostB}, 1)`} />
+      </Box>
```

Comment:
```suggestion
      <Stack vertical>
        <Stack.Item>
          <strong># Loaded Lights:</strong>
          {' '}{ lights }
        </Stack.Item>
        <Stack.Item>
          Color Tuning:
          {' '}
          <Knob
            inline
            minValue={20}
            maxValue={255}
            value={hostR}
            color="red"
            format={value => "R:" + value}
            onDrag={(e, value) => act('tune_hue', {
              hue: "R",
              output: value,
            })}
          />
          <Knob
            inline
            minValue={20}
            maxValue={255}
            value={hostG}
            color="green"
            format={value => "G:" + value}
            onDrag={(e, value) => act('tune_hue', {
              hue: "G",
              output: value,
            })}
          />
          <Knob
            inline
            minValue={20}
            maxValue={255}
            value={hostB}
            color="blue"
            format={value => "B:" + value}
            onDrag={(e, value) => act('tune_hue', {
              hue: "B",
              output: value,
            })}
          />
          {' '}
          <ColorBox color={`rgba(${hostR}, ${hostG}, ${hostB}, 1)`} />
        </Stack.Item>
      </Stack>
```
Prefer using a `Stack` here for better spacing (would need to be imported as well).

Reply:
Stackified in f7358dc.